### ISSUE TITLE
Add JUnit5 to Maven Central repo

### ISSUE THREAD

**Issue by peter-lyons-kehl** [junit-team/junit5#1526]
Dear JUnit team,

Thank you for JUnit. And for new features of JUnit5. Could you add it to Maven Central repo, please.

Building JUnit5 and plugging it to my project seems too complicated. Currently my build fails: "Could not find artifact junit:junit:jar:5.2.0 in central (https://repo.maven.apache.org/maven2)". JUnit5 is listed under http://repo1.maven.org/maven2/org/junit/, but from 2016 - old.

**Comment by smoyer64** [comment]
JUnit 5 has been in Maven Central since the alpha (prototype) release but no longer shares the org.junit groupId.  There are a lot of different artifacts but, as a Maven user I typically only use three or four of them.  You're going to need to learn its architecture a bit - here are the packages available in Maven Central - https://junit.org/junit5/docs/current/user-guide/#dependency-metadata

**Comment by nicolaiparlog** [comment]
TL;DR, use these artifacts:

```xml
<!-- this is all you need to write tests with JUnit Jupiter -->
<dependency>
	<groupId>org.junit.jupiter</groupId>
	<artifactId>junit-jupiter-api</artifactId>
	<version>${junit-jupiter-version}</version>
	<scope>test</scope>
</dependency>
<!-- this dependency is needed to create parameterized tests -->
<dependency>
	<groupId>org.junit.jupiter</groupId>
	<artifactId>junit-jupiter-params</artifactId>
	<version>${junit-jupiter-version}</version>
	<scope>test</scope>
</dependency>
<!-- contains the engine that actually runs the Jupiter-tests -->
<dependency>
	<groupId>org.junit.jupiter</groupId>
	<artifactId>junit-jupiter-engine</artifactId>
	<version>${junit-jupiter-version}</version>
	<scope>test</scope>
</dependency>
```

And define:

```xml
<properties>
	<junit-jupiter-version>5.2.0</junit-jupiter-version>
</properties>
```

Have, fun writing tests! :smiley:

If the user guide isn't for you, I wrote [a post about JUnit 5 setup](https://blog.codefx.org/libraries/junit-5-setup/).

**Comment by saurabhChaubey** [comment]
still not working for me
